packages/react/src/primitives/Expander/Expander.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import { Root } from '@radix-ui/react-accordion';
+
+import { Primitive } from '../types/view';
+import { ExpanderProps } from '../types/expander';
+import { ComponentClassNames } from '../shared/constants';
+import { splitPrimitiveProps } from '../shared/styleUtils';
+
+export const Expander: Primitive<ExpanderProps, typeof Root> = ({
+  children,
+  className,
+  defaultValue,
+  isCollapsible,
+  onChange,
+  // It is not in use but remove it from rest to avoid type errors
+  onValueChange,
+  testId,
+  type = 'single',
+  value,
+  ..._rest
+}) => {
+  const { rest } = splitPrimitiveProps(_rest);
+
+  const expander =
```

Comment:
is the `isCollapsible` property the only difference between these two? I wonder if there's a simpler way to write this conditional render 🤔

Reply:
No, it will start crying